### DISCUSSION TITLE
Vehicle control with interp1d

### DISCUSSION THREAD

**Discussion by carlosxsm** [petercorke/robotics-toolbox-python#250] [Q&A]
Hello,
I have been using this RTB for several days now. But I still cannot find a way to make the control part for the Vehicle work with the interpolator. Would you please give some more details about it?

From the source it reads:
Control can be: * a constant tuple as the control inputs to the vehicle * a function called as ``f(vehicle, t, x)`` that returns a tuple * an interpolator called as f(t) that returns a tuple, see SciPy interp1d * a driver agent, subclass of :func:`VehicleDriver` Example: .. runblock:: pycon >>> bike = Bicycle() >>> bike.control = RandomPath(10) >>> print(bike)

How would I correctly make the call to that function?
I keep getting error.

Thanks

**Comment by petercorke** [comment]
Mobile robotics isn't finished yet, it's simply a quick port of MATLAB code with no testing.  It's my project for next month so watch the `future` branch.
